docs/docs/noir/concepts/generics.md

```diff
@@ -18,6 +18,23 @@ fn id<T>(x: T) -> T  {
 }
 ```
 
+## Numeric Generics
+
+Numeric generics are specified using the `let NAME: TYPE` syntax, and are
+required when a generic `N` is used for the size of an array (`[T; N]`) or string (`str<N>`):
+
```

Comment:
This section in the docs has since been added after the "In Structs" section we can merge them or edit/remove one but we shouldn't have two